bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -501,11 +496,20 @@ private void mutateRow(Mutation mutation, RowMutation rowMutation, String type)
   }
 
   /** {@inheritDoc} */
-  @Override
-  public void mutateRow(RowMutations rowMutations) throws IOException {
+  protected Result mutateRowBase(RowMutations rowMutations) throws IOException {
     LOG.trace("mutateRow(RowMutation)");
+
+    if (rowMutations.getMutations().size() == 1) {
+      Mutation onlyMutation = rowMutations.getMutations().get(0);
+      if (onlyMutation instanceof Append) {
```

Comment:
Can we add a comment saying that Bigtable only supports single append and increment, and line 516 is gonna reject the case where there are multiple append / increment ?

Reply:
done